### PR TITLE
Creates Wordpress cheat sheet

### PR DESCRIPTION
**Pull request by aleksandar-todorovic**
**What does your Instant Answer do?**
Makes a Wordpress cheatsheet.

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://codex.wordpress.org/Keyboard_Shortcuts

**Are there any other alternative (better) data sources?**
The one I used is located on a wordpress.org domain, so I'm assuming the answer is no.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Web developers

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
https://duck.co/ideas/idea/6311/wordpress-cheat-sheet

**Are you having any problems? Do you need our help with anything?**
The first category is way longer then the rest of the categories, so it kind of doesn't look good ecstatically.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

screenshot from 2015-08-29 18 52 15

**NOTE** I'm submitting this from my organization profile since I made a mistake of not creating a separate branch _before_ I developed my first pull request.

IA Page: https://duck.co/ia/view/wordpress_cheat_sheet

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
@aleksandar-todorovic Thanks a lot for this! We'll review and get back to you shortly :)

File: share/goodie/cheat_sheets/json/wordpress.json [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+            "key": "[Ctrl] [i]"
+        }, {
+            "val": "Underline",
+            "key": "[Ctrl] [u]"
+        }, {
+            "val": "Insert/edit link",
+            "key": "[Ctrl] [k]"
+        }, {
+            "val": "Check spelling",
+            "key": "[Alt] [Shift] [n]"
```
Comment (mattr555) [inline share/goodie/cheat_sheets/json/wordpress.json]: You have `[Alt] [Shift] [n]` for "Check spelling" and again for "Exit visual mode."

**Comment by mattr555** [comment]
Looks great, thank you! :+1:  I checked against the alt+shift+h menu in wordpress itself, so this is accurate. I unfortunately have a history that involves wordpress :/

The only other thing I have to say is that the editor shortcut section is kind of big compared to the others. I wonder if it can be broken up into different sections?

File: share/goodie/cheat_sheets/json/wordpress.json (hunk 1 above)
Comment (aleksandar-todorovic) [inline share/goodie/cheat_sheets/json/wordpress.json]: I just checked it out and it turns out that the second one ("Exit visual mode") should (and will) be deleted all together. It remained there from the vim cheat sheet which I copied when I started making this.

**Comment by aleksandar-todorovic** [comment]
@mattr555 In a web source I used, This category is split into two subheadings: Those involving Ctrl key combination and those involving Shift + Alt key combination, but they're both applicable to the Wordpress editor, so I have no idea how to split them up but make sure it's clear that these keyboard shortcuts are only relevant to the Wordpress editor.

**Comment by mattr555** [comment]
@aleksandar-todorovic Ok, it's not a big deal either way. I think this is good to go other than the other comment I had about `alt-shift-n`, I didn't see a reference for it in the sources.

**Comment by aleksandar-todorovic** [comment]
@mattr555 It's removed now.

**Comment by mattr555** [comment]
Great, LGTM! We can merge after @moollaza signs off :memo:

File: share/goodie/cheat_sheets/json/wordpress.json [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            "val": "Unapprove checked comments",
+            "key": "[Shift] [u]"
+        }, {
+            "val": "Move checked comments to trash",
+            "key": "[Shift] [t]"
+        }, {
+            "val": "Restore checked comments from trash",
+            "key": "[Shift] [z]"
+        }],
+        "Editor Shortcuts": [{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/wordpress.json]: Let's split this in two. `Editor Shorcuts` and `Editor Shortcuts (continued)` is fine.

**Comment by moollaza** [comment]
@aleksandar-todorovic great work, I've left 1 comments about splitting the editor commands in two. After that this is good to be merged!

**Comment by jsstrn** [comment]
Might be worth mentioning that [WordPress](https://wordpress.org/) should be spelt with a capital 'P'.

**Comment by jagtalon** [comment]
@jsstrn sweet! Please do https://github.com/duckduckgo/zeroclickinfo-goodies/pull/1469#discussion_r38264779 and it should be good to go!

**Comment by aleksandar-todorovic** [comment]
@jagtalon Done! Should be ready to merge now.

**Comment by mattr555** [comment]
@aleksandar-todorovic Sorry, _realllly_ nitpicky, but could we move the `Ctrl-k` command to the first set of commands? If we're going to break it somewhere it makes sense to do it there.

**Comment by aleksandar-todorovic** [comment]
@mattr555 Done!

**Comment by mattr555** [comment]
:+1: :tada: Merging this. Thanks a lot!

**Comment by jagtalon** [comment]
Woot!

**Comment by moollaza** [comment]
@aleksandar-todorovic congrats! It's live: https://duckduckgo.com/?q=wordpress+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**